### PR TITLE
Added Test Completion output and nvsmi output to verbose

### PR DESCRIPTION
**Pull request by ElizabethFatade**
Added successful test completion to health checks output and log files for each nhc test. In addition to that I have also added the output of nvidia-smi to the health log of the health checks when the -v (verbose) flag is added.

### CODE REVIEW

File: customTests/azure_gpu_app_clocks.nhc [hunk 1]
```diff
@@ -42,5 +42,7 @@ function check_app_gpu_clocks() {
          dbg "GPU Id $i: max application GPU clocks are already set, GPU memory is  ${gpu_freq_out_line[0]} MHz and GPU graphics is ${gpu_freq_out_line[2]} MHz"
       fi
 done
+
+pass 0 "$FUNCNAME: Application GPU clock frequencies set to max"
 return 0
 }
```
Comment (rafsalas19) [inline customTests/azure_gpu_app_clocks.nhc]: Can you add an empty line here. (This was probably carried over from before)

File: customTests/azure_cpu_stream.nhc [hunk 2]
```diff
@@ -3,6 +3,8 @@
 # Run Stream
 
 EXE_DIR=$AZ_NHC_ROOT/bin
+source customTests/azure_common.nhc
```
Comment (rafsalas19) [inline customTests/azure_cpu_stream.nhc]: Please use this instead: /etc/nhc/scripts/azure_common.nhc

When we build the docker container this is where the file will/should reside. This is probably a good time to look into setting up a test ACR so we can build and publish test docker images.

File: customTests/azure_gpu_amd_xgmi.nhc [hunk 3]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 source /etc/nhc/scripts/azure_common.nhc
+source customTests/azure_common.nhc
```
Comment (rafsalas19) [inline customTests/azure_gpu_amd_xgmi.nhc]: you can delete this line as we grab it from above^

File: customTests/azure_gpu_bandwidth.nhc [hunk 4]
```diff
@@ -7,6 +7,8 @@
 # - device to device bandwidth
 
 source /etc/nhc/scripts/azure_common.nhc
+source "customTests/azure_common.nhc"
```
Comment (rafsalas19) [inline customTests/azure_gpu_bandwidth.nhc]: same comment as above

File: customTests/azure_gpu_count.nhc [hunk 5]
```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+
+source customTests/azure_common.nhc
```
Comment (rafsalas19) [inline customTests/azure_gpu_count.nhc]: same comment

File: customTests/azure_gpu_app_clocks.nhc [hunk 6]
```diff
@@ -3,7 +3,7 @@
 # Check if application GPU clock frequencies are set to their maximum values, if not will attempt to set them.
 
 GPU_QUERY_CLOCKS="clocks.max.memory,clocks.applications.memory,clocks.max.graphics,clocks.applications.graphics"
-
+source customTests/azure_common.nhc
```
Comment (rafsalas19) [inline customTests/azure_gpu_app_clocks.nhc]: Please use this instead: /etc/nhc/scripts/azure_common.nhc

File: customTests/azure_gpu_ecc.nhc [hunk 7]
```diff
@@ -4,6 +4,7 @@
 
 GPU_REMAPPED_ROWS_QUERY="remapped_rows.pending,remapped_rows.failure,remapped_rows.uncorrectable"
 GPU_QUERY="ecc.errors.uncorrected.volatile.sram,ecc.errors.uncorrected.aggregate.sram,ecc.errors.uncorrected.volatile.dram,ecc.errors.uncorrected.aggregate.dram,ecc.errors.corrected.volatile.sram,ecc.errors.corrected.aggregate.sram,ecc.errors.corrected.volatile.dram,ecc.errors.corrected.aggregate.dram"
+source customTests/azure_common.nhc
```
Comment (rafsalas19) [inline customTests/azure_gpu_ecc.nhc]: Please use this instead: /etc/nhc/scripts/azure_common.nhc

File: customTests/azure_gpu_nvlink.nhc [hunk 8]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+source customTests/azure_common.nhc
```
Comment (rafsalas19) [inline customTests/azure_gpu_nvlink.nhc]: Please use this instead: /etc/nhc/scripts/azure_common.nhc

File: customTests/azure_gpu_persistence.nhc [hunk 9]
```diff
@@ -3,7 +3,7 @@
 # Check GPU persistence mode, if not enabled, attempt to enable.
 
 PERSISTENCE_GPU_QUERY="persistence_mode"
-
+source customTests/azure_common.nhc
```
Comment (rafsalas19) [inline customTests/azure_gpu_persistence.nhc]: Please use this instead: /etc/nhc/scripts/azure_common.nhc

File: customTests/azure_gpu_app_clocks.nhc (hunk 1 above)
Comment (ElizabethFatade) [reply customTests/azure_gpu_app_clocks.nhc]: Done

File: customTests/azure_cpu_stream.nhc (hunk 2 above)
Comment (ElizabethFatade) [reply customTests/azure_cpu_stream.nhc]: I have made changes on the different test files using /etc/nhc/scripts/azure_common.nhc now

**Review by rafsalas19 — APPROVED** [review]

**Pull request merged**